app/controllers/AskSecondaryWarehouseInReturnController.scala

```diff
@@ -47,26 +48,25 @@ class AskSecondaryWarehouseInReturnController @Inject()(
 
   val form = formProvider()
 
-  def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData) {
+  def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData andThen checkSub) {
```

Comment:
Was there a reason why this was missing requireData initially? Was it deliberate?

Reply:
I think this page was done before navigation and didn't include required data so you could access this page without having pervious answers